### PR TITLE
Added support for tuya  TS0601_TZE204_wktrysab 8-gang wall switch touch board

### PR DESCRIPTION
**Pull request by rhakbari**

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -811,6 +811,63 @@ const definitions: DefinitionWithExtend[] = [
         toZigbee: [],
         exposes: [e.gas(), e.tamper()],
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: 'Melery',
```
Comment (Koenkk) [inline src/devices/tuya.ts:818]: This manufaturerName is unusual, could you provide the data/database.db entry of this device?

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -811,6 +811,63 @@ const definitions: DefinitionWithExtend[] = [
...
+                [2, 'state_l2', tuya.valueConverter.onOff],
+                [3, 'state_l3', tuya.valueConverter.onOff],
+                [4, 'state_l4', tuya.valueConverter.onOff],
+                [5, 'state_l5', tuya.valueConverter.onOff],
+                [6, 'state_l6', tuya.valueConverter.onOff],
+                [0x65, 'state_l7', tuya.valueConverter.onOff],
+                [0x66, 'state_l8', tuya.valueConverter.onOff],
+                [0x67, 'null', null],
+                [0x68, 'null', null],
+                [0x69, 'null', null],
```
Comment (Koenkk) [inline src/devices/tuya.ts:854]: Why are these needed?

**Comment by rhakbari** [comment]
Could you please review the PR for any additional changes that might be needed? If everything looks good, kindly proceed to merge it. Thank you!

**Comment by Koenkk** [comment]
Thanks!

Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `WLS098-ZIGBEE.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

**Comment by rhakbari** [comment]
> Thanks!
> 
> Could you also submit a picture for the docs? This can be done by:
> 
> 1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
> 2. Go to the `public/images/devices` directory, _Add file_ -> _Upload files_
> 3. Upload the files and press _Commit changes_
> 4. Press _Contribute_ -> _Open pull request_ -> update title/description -> _Create pull request_
> 
> Make sure that:
> 
> * The filename is `WLS098-ZIGBEE.png`
> * The size is 512x512
> * The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

Done
